### PR TITLE
fix: disabled FFMPEG video decoder plugin that was causing crashes

### PR DESCRIPTION
**Pull request by Kinerius**
## What does this PR change?

Fixes a random crash caused by FFMPEG decoder by disabling the plugin temporaly

Also fixed Unity forced SSL and port

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer-desktop/Assets/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs [hunk 1]
```diff
@@ -46,9 +46,16 @@ protected override void InitializeCommunication()
             //              IntegrationTestSuite_Legacy base class.
             if (!Configuration.EnvironmentSettings.RUNNING_TESTS)
             {
+                var withSSL = true;
                 int startPort = CommandLineParserUtils.startPort;
+                
+                #if UNITY_EDITOR
+                withSSL = false;
```
Comment (nlorux) [inline unity-renderer-desktop/Assets/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs:53]: this makes sense.. however you could remove `Web Socket SSL` from `DebugConfigComponent` ?

**Review by nlorux — CHANGES_REQUESTED** [review]

**Comment by nlorux** [comment]
since video streaming is now disabled because it was crashing the app while running through the metaverse, it would be good to confirm this merge with @olavra because before this changes i was able to see at least one video. I know it this issue has to be fixed, but perhaps it can be done in this PR

File: unity-renderer-desktop/Assets/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs (hunk 1 above)
Comment (kuruk-mm) [inline unity-renderer-desktop/Assets/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs:53]:
```suggestion
                withSSL = DebugConfigComponent.i.webSocketSSL;
```

Can you make this change?

**Review by kuruk-mm — CHANGES_REQUESTED** [review]

File: unity-renderer-desktop/Assets/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs (hunk 1 above)
Comment (Kinerius) [reply unity-renderer-desktop/Assets/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs]: Yes! totally agree

**Review by pravusjif — APPROVED** [review]
With this build the never-ending loading scenes issue doesn't happen so it must be provoked by the problems with the FFMPEG plugin, nice catch

**Pull request merged**